actions/actions.py

```diff
@@ -501,22 +504,18 @@ def run(
         for intent in first_intent_names:
             logger.debug(intent)
             logger.debug(entities)
-            buttons.append(
-                {
-                    "title": self.get_button_title(intent, entities),
-                    "payload": "/{}{}".format(intent, entities_json),
-                }
-            )
+            if len(intent.split("/")) > 1:
+                message = self.get_button_title(intent, entities)
+                buttons.append({"title": message, "payload": message})
+            else:
+                buttons.append(
+                    {
+                        "title": self.get_button_title(intent, entities),
```

Comment:
this "title" is actually the same as L 508-509 but it doesn't look so, let's pull out 508 outside of if/else and use it in both titles